gui/pages/user_add.go

```diff
@@ -76,15 +76,15 @@ func NewUserAddPage(controller Controller, model *model.SystemInstall) (Page, er
 
 	// Name
 	page.name, page.nameWarning, err = page.setSimilarWidgets(utils.Locale.Get("User Name"),
-		utils.Locale.Get("Must start with letter. Can use numbers, hyphens and underscores. Max %d characters.", user.MaxUsernameLength),
+		utils.Locale.Get("Must start with letter. Can use numbers, commas, - and _. Max %d characters.", user.MaxUsernameLength),
 		user.MaxUsernameLength)
 	if err != nil {
 		return nil, err
 	}
 
 	// Login
 	page.login, page.loginWarning, err = page.setSimilarWidgets(utils.Locale.Get("Login")+" *",
-		utils.Locale.Get("Must start with letter. Can use numbers, hyphens and underscores. Max %d characters.", user.MaxLoginLength),
+		utils.Locale.Get("Must start with letter. Can use numbers, - and _. Max %d characters.", user.MaxLoginLength),
```

Comment:
Same here:
"Must start with letter. Can use numbers, '-' and '_'. Max %d characters."